client/lib/analytics/survicate.js

```diff
@@ -58,8 +75,17 @@ export function addSurvicate() {
 		return;
 	}
 
-	if ( survicateScriptLoaded || ! mayWeLoadSurvicateScript() ) {
-		survicateDebug( 'Not loading Survicate script' );
+	if ( survicateScriptLoaded && mayWeLoadSurvicateScript() ) {
```

Comment:
This condition is odd. Why do we need to check `mayWeLoadSurvicateScript()` if `survicateScriptLoaded` is `true`? Won't the `! mayWeLoadSurvicateScript()` check below prevent `survicateScriptLoaded` from becoming `true` ever?

If what I'm saying is correct, we may just combine this check with the one that debugs "already loaded".

Reply:
Good catch, this didn't end up well after the second approach. Yeah, let's combine them.